sources/entities/entities.cpp

```diff
@@ -77,6 +77,9 @@ void Entities::load() {
                 QVariantMap           map = type[k].toMap();
                 QObject *             obj = Integrations::getInstance()->get(map.value("integration").toString());
                 IntegrationInterface *integration = qobject_cast<IntegrationInterface *>(obj);
+                if (!integration) {
+                    qCWarning(CLASS_LC()) << "integration pointer is null" << map.value("integration").toString();
+                }
```

Comment:
If integration is null we should probably not add it to m_supportedEntities which will most likely produce a runtime error later on.
@martonborzak or is there anything I'm missing? I.e. can an entity have no integration?

Reply:
Entities usually have integrations. Null pointer will crash the app.